jwt-authorizer/src/layer.rs

```diff
@@ -414,3 +483,36 @@ where
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::{IntoLayer, JwtAuthorizer};
+
+    #[tokio::test]
+    async fn jwt_auth_to_layer() {
+        let auth1: JwtAuthorizer = JwtAuthorizer::from_secret("aaa");
+        let layer = auth1.into_layer().await;
+        assert!(layer.is_ok());
+    }
+
+    #[tokio::test]
+    async fn vec_to_layer() {
+        let auth1: JwtAuthorizer = JwtAuthorizer::from_secret("aaa");
+        let auth2: JwtAuthorizer = JwtAuthorizer::from_secret("bbb");
+        let av = vec![auth1, auth2];
+        let layer = av.into_layer().await;
+        assert!(layer.is_ok());
+    }
+
+    #[tokio::test]
+    async fn vec_to_layer_errors() {
+        let auth1: JwtAuthorizer = JwtAuthorizer::from_ec_pem("aaa");
+        let auth2: JwtAuthorizer = JwtAuthorizer::from_ed_pem("bbb");
+        let av = vec![auth1, auth2];
+        let layer = av.into_layer().await;
+        assert!(layer.is_err());
+        if let Err(err) = layer {
+            assert_eq!(err.to_string(), "No such file or directory (os error 2)");
+        }
```

Comment:
```suggestion
        let err = av.into_layer().await.unwrap_err();
         assert_eq!(err.to_string(), "No such file or directory (os error 2)");
```
Both shorter and will print the Ok value if the result is not an error